theme/static/style/style.css

```diff
@@ -98,6 +98,113 @@ ul.topnav li [name="q"] {
     border: 0;
 }
 
+.topnav{
+    padding: 10px;
+    overflow: hidden;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0, .3);
+    z-index: 1;
+}
+
+.topnav ul{
+    list-style-type: none;
+    text-align: right;
+}
+
+.topnav li{
+    display: inline-block;
+    margin-left: 15px;
+    vertical-align: top;
```

Comment:
This property puts the navigation bar textbox out of center vertically and does not affect any other element. Remove it or modify it to have useful effects.